### PR TITLE
Relax dependency on `httparty` gem

### PR DESCRIPTION
**Pull request by johnnyshields**
Includes the following gem cleanup items:
- Relax version dependency on httparty, RSpec and Jeweler gems
- Gemfile.lock should be gitignored
- Add Ruby 2.1.0 to Travis

### CODE REVIEW

**Comment by johnnyshields** [comment]
@jtadeulopes pinging again, please merge.

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,4 @@ spec/dummy/tmp/
 .rspec
 .rvmrc
 .rbenv-version
+Gemfile.lock
```
Comment (jtadeulopes) [inline .gitignore]: Why?

Comment (johnnyshields) [inline .gitignore]: Please read this article. http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

TDLR; Gemfile.lock should only be checked in for projects, not gems.

Comment (jtadeulopes) [inline .gitignore]: Nice. Thank you!

Today, I'll release the new version.

**Comment by jtadeulopes** [comment]
Successfully registered gem: googl (0.7.0)

**Pull request merged**